lib/src/operations/ledger_operations.dart

```diff
@@ -0,0 +1,41 @@
+import 'dart:typed_data';
+
+import 'package:ledger_cardano/src/operations/cardano_ledger_operation.dart';
+import 'package:ledger_flutter/ledger_flutter.dart';
+
+class SendOperation extends LedgerOperation<ByteDataReader> {
+  final int ins;
+  final int p1;
+  final int p2;
+  final Uint8List data;
+  final bool expectResponseLength;
```

Comment:
This should probably be renamed to `prependDataLength`. Current name is not correctly reflecting its purpose.

Reply:
done